cypress-tests/cypress/e2e/PaymentUtils/Adyen.js

```diff
@@ -69,6 +69,23 @@ export const connectorDetails = {
         },
       },
     },
+    PaymentIntentOffSession: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
```

Comment:
Remove payment method details from the intent Call. Because i made the changes in intentflow. Currently it wont pick it from the config file. After my change it will pick from config file. That time if PM is present then the state wont go as "requires_payment_method"